rj_gameplay/rj_gameplay/tactic/wall_tactic.py

```diff
@@ -18,30 +18,33 @@
 from stp.utils.constants import RobotConstants, BallConstants
 import stp.global_parameters as global_parameters
 
+MIN_WALL_RAD = None
```

Comment:
the only changes to this file from what's in #1661 are from `make pretty-lines`, skip this too